### DISCUSSION TITLE
Pen input on card surfaces

### DISCUSSION THREAD

**Discussion by jordanh** [ParabolInc/parabol#5957] [New features]
## Issue - Enhancement

We hear input that folks wish they could draw or somehow the meeting input could be more visually creative (#3923). Folks also want to add images to cards (#3957).

What if we allowed simple sketching / pen input? Using our meetings on a tablet would be _a lot_ more engaging.

**Comment by jordanh** [comment]
First validation!

[A user](https://app.hubspot.com/contacts/3888472/contact/13216451) writes: 

> One feature request! I'd love to have cards that I can draw on - prior to my team going partially remote, we had a lot of retro activities that involved drawing, and I'd love to be able to integrate those back in.

**Comment by moskyb** [comment]
👋 hi! I'm "a user" referred to above.

A common activity we'd do prior to using parabol was to draw how we feel about the last sprint on a post it note and talk about it. We also archived everyone's past cards, so that we could track trends in how we feel about our work, though I suspect that this aspect (at least the archival stage) would be difficult to replicate in Parabol. 

We'd also often do normal retro activities (Mad/Glad/Sad, Sailboat etc, all the usual suspects), but augment things we write with doodles and such - it makes our cards feel more personal, I think. It also allowed me to better remember why I wrote something on a card - I have some memory/attention issues, and putting more emotion into a card by drawing on it helped me connect what I put on the card with how I actually feel about something.

One potential downside I can see is that drawing on a laptop usually sucks a bit - perhaps this could be solved with a mobile app or something, though that's a pretty big lift for just pen input.

**Comment by drewhousman1010** [comment]
A [user ](https://app.hubspot.com/contacts/3888472/contact/20672801) 🔒 writes:

> I would love to see some whiteboard functionality (Like Miro) as that is the only piece missing that causes me to use aforementioned platform.

**Comment by jordanh** [comment]
See also: #4516 (for examples of more input types on Reflection cards)

[a comment by an automated account is not shown]
